core/server/worker/src/main/java/alluxio/worker/netty/NettyDataServer.java

```diff
@@ -170,8 +163,8 @@ private ServerBootstrap createBootstrapOfType(final ChannelType type) {
     final EventLoopGroup workerGroup =
         NettyUtils.createEventLoop(type, workerThreadCount, "data-server-worker-%d", false);
```

Comment:
Do both the network server and the domain socket server both use this "data-server-worker-%d" name?